src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
1. Is this related to auth0? If not, it would be ideal if this was in a different PR
2. I'm assuming this is copied from create-react-app? 
  - Do their assumptions they're making (if any) also hold true in this codebase?
  - Can you add a comment indicating where this was sourced from